### ISSUE TITLE
Unknown ED name (faction name) in in resource EddiDataDefinitions.Properties.Powers

### ISSUE THREAD

**Issue by Tkael** [EDCD/EDDI#1372]
## EDDI version in which issue is found
3.4.1-rc1	

## VoiceAttack version in which issue is found (as applicable)
N/A

## Steps to reproduce
1. Target a ship not associated with a power
2. Review the log output

## Expected
EDDI doesn't confuse powers and factions

## Observed
2019-06-22T19:04:32 [Info] ResourceBasedLocalizedEDName:FromEDName Unknown ED name mobiuscolonialrepublicnavy in resource EddiDataDefinitions.Properties.Powers 
2019-06-22T19:04:33 [Info] ResourceBasedLocalizedEDName:FromEDName Unknown ED name nantariatesstateholdings in resource EddiDataDefinitions.Properties.Powers 
2019-06-22T19:04:34 [Info] ResourceBasedLocalizedEDName:FromEDName Unknown ED name labourofanhualm in resource EddiDataDefinitions.Properties.Powers 

## Investigation
CrimeMonitor.cs line 251 attempts to match the faction name to a power using the Power.cs `FromEDName` method. The Power.cs `FromEDName` method calls the ResourceBasedLocalizedEDName `FromEDName` method, and the `FromEDName` method logs an unknown power whenever it fails to return a match.

**Comment by Hoodathunk** [comment]
When a ship is targeted, the `faction` is either a minor faction or a power. I agree that blindly using `FromEDName` to test for a power is resulting in [Info] reports that should be screened out. How about we modify the override with...
```        public new static Power FromEDName(string edName)
        {
            if (edName == null)
            {
                return null;
            }

            string tidiedName = edName.ToLowerInvariant().Replace(" ", "").Replace(".", "").Replace("-", "");
            Power result = AllOfThem.FirstOrDefault(v => v.edname == tidiedName);
            if (result == null) { return Power.None;  }
            return result;
        }
```

**Comment by Tkael** [comment]
That change looks like it would be an acceptable alternative. Cleaner, really.

**Comment by Tkael** [comment]
Closing in favor of #1375.

**Comment by Hoodathunk** [comment]
Fixed by PR #1375.
